.github/workflows/build-website.yml

```diff
@@ -6,6 +6,11 @@ on:
     schedule:
         - cron: '0 9,18 * * *'
 
+concurrency:
+    group: website-deployment
+    # TODO: Enable cancel-in-progress once we are deploying per commit?
```

Comment:
I'd let it finish – stopping the deployment halfway through or when the `apply-update.sh` is running sounds like trouble. If we push 5 commits in rapid progression, though, perhaps this could only run for the first and the fifth, but not for anything in between?

Reply:
> I'd let it finish – stopping the deployment halfway through or when the apply-update.sh is running sounds like trouble.

Good point.

> If we push 5 commits in rapid progression, though, perhaps this could only run for the first and the fifth, but not for anything in between?

That's my understanding: There can be one workflow instance active and one pending. If another becomes pending, it replaces the previous pending workflow.